WordPress/Classes/ViewRelated/Site Creation/TitleSubtitleHeaderDataCoordinator.swift

```diff
@@ -27,23 +27,27 @@ final class SiteCreationDataCoordinator: NSObject, UITableViewDataSource, UITabl
     }
 
     func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        let title = UILabel(frame: .zero)
-        title.translatesAutoresizingMaskIntoConstraints = false
-        title.text = headerData.title
-        title.textAlignment = .center
-        title.numberOfLines = 0
-
-        let subtitle = UILabel(frame: .zero)
-        subtitle.translatesAutoresizingMaskIntoConstraints = false
-        subtitle.text = headerData.subtitle
-        subtitle.textAlignment = .center
-        subtitle.numberOfLines = 0
-
-        let stackView = UIStackView(arrangedSubviews: [title, subtitle])
-        stackView.axis = .vertical
-        stackView.spacing = 20
-
-        return stackView
+        let header = TitleSubtitleHeader(frame: .zero)
+        header.setTitle(headerData.title)
+        header.setSubtitle(headerData.subtitle)
+        return header
+//        let title = UILabel(frame: .zero)
```

Comment:
Can we remove the commented-out code?

Reply:
I've removed the whole class. It should not have been part of the PR